pennylane_lightning/src/util/Util.hpp

```diff
@@ -36,6 +36,13 @@
 #define USE_CBLAS 1
 #else
 #define USE_CBLAS 0
+
+#define CblasTrans 0
+#define CblasNoTrans 0
+#define CblasRowMajor 0
```

Comment:
Since these flags are used to select logical options, defining them with an `enum` is a safer move. This can be placed in an anonymous namespace to enable visibility within the file only.

Reply:
[This article by Sy Brand](https://blog.tartanllama.xyz/if-constexpr/) who works on Microsoft's MSVC compiler has some good examples of doing conditional compilation for different operating systems.